src/content/changes/highlights/2022-11-10.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2022-11-10T10:00:00
+title: Highlights for the week ending November 10, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## User interfaces
+- [kubectl-gs v2.28.1](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.28.1) avoids panic in get nodepools when node pool is lacking the release version label and disables kubectl gs template nodepool command for Cluster API (CAPI) based workload Clusters.
+- [happa v1.53.0](https://github.com/giantswarm/happa/releases/tag/v1.53.0) adds support for read-only cluster and nodepool management for Cluster API provider AWS (CAPA), Update cluster creation message for GCP (Google Cloud Platform) to better reflect actual cluster creation duration and provides links for installed apps to allow navigating to the app's catalog page and catalog item details page
+## Apps
+- [dex v1.30.1](https://github.com/giantswarm/dex-app/releases/tag/v1.30.1) updates dex to v2.35.3
+- [flux-app](https://github.com/giantswarm/flux-app) version [v0.18.2](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0182---2022-11-10). This release fixes VerticalPodAutoscaler issues. VPAs are now correctly configured when enabled with a feature flag in `values.yaml`.
+- [linkerd-multicluster](https://github.com/giantswarm/linkerd-multicluster-app) version [v0.9.0](https://github.com/giantswarm/linkerd-multicluster-app/blob/main/CHANGELOG.md#090---2022-11-07) Update linkerd to [stable-2.12.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.2). In this release the app has been renamed as `linkerd-multicluster` following upstream's naming convention.
+- [linkerd-multicluster-link](https://github.com/giantswarm/linkerd-multicluster-link-app) version [v0.9.0](https://github.com/giantswarm/linkerd-multicluster-link-app/blob/main/CHANGELOG.md#090---2022-11-07) Update linkerd to [stable-2.12.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.2). In this release the app has been renamed as `linkerd-multicluster-link` following upstream's naming convention. Also, the value `target.name` has been deprecated and replaced with `targetClusterName`.
+- [prometheus-operator-app](https://github.com/giantswarm/prometheus-operator-app) version [v2.0.2](https://github.com/giantswarm/prometheus-operator-app/releases/tag/v2.0.2). Update prometheus-operator to [v0.54.0](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.54.0). This release include breaking changes and `values.yaml` need to be adapated, see [update-changes](https://github.com/giantswarm/prometheus-operator-app/blob/v2.0.0/changelog/23.x_32.x.md.).
+- [prometheus-operator-crd](https://github.com/giantswarm/prometheus-operator-crd) version [v2.0.1](https://github.com/giantswarm/prometheus-operator-crd/releases/tag/v2.0.1). Update CRD for prometheus-operator.
```

Comment:
```suggestion
## Apps

- [dex v1.30.1](https://github.com/giantswarm/dex-app/releases/tag/v1.30.1) updates dex to v2.35.3
- [falco-app](https://github.com/giantswarm/falco-app) version [v0.4.0](https://github.com/giantswarm/falco-app/blob/master/CHANGELOG.md#040---2022-11-09) updates to Falco v0.33.0, falco-exporter v0.8.0, and falcosidekick v2.26.0.
- [flux-app](https://github.com/giantswarm/flux-app) version [v0.18.2](https://github.com/giantswarm/flux-app/blob/master/CHANGELOG.md#0182---2022-11-10). This release fixes VerticalPodAutoscaler issues. VPAs are now correctly configured when enabled with a feature flag in `values.yaml`.
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.11.8](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0118---2022-11-09) fixes an issue where kyverno-app would fail to install on clusters using Cilium CNI.
- [linkerd-multicluster](https://github.com/giantswarm/linkerd-multicluster-app) version [v0.9.0](https://github.com/giantswarm/linkerd-multicluster-app/blob/main/CHANGELOG.md#090---2022-11-07) Update linkerd to [stable-2.12.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.2). In this release the app has been renamed as `linkerd-multicluster` following upstream's naming convention.
- [linkerd-multicluster-link](https://github.com/giantswarm/linkerd-multicluster-link-app) version [v0.9.0](https://github.com/giantswarm/linkerd-multicluster-link-app/blob/main/CHANGELOG.md#090---2022-11-07) Update linkerd to [stable-2.12.2](https://github.com/linkerd/linkerd2/releases/tag/stable-2.12.2). In this release the app has been renamed as `linkerd-multicluster-link` following upstream's naming convention. Also, the value `target.name` has been deprecated and replaced with `targetClusterName`.
- [prometheus-operator-app](https://github.com/giantswarm/prometheus-operator-app) version [v2.0.2](https://github.com/giantswarm/prometheus-operator-app/releases/tag/v2.0.2). Update prometheus-operator to [v0.54.0](https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.54.0). This release include breaking changes and `values.yaml` need to be adapated, see [update-changes](https://github.com/giantswarm/prometheus-operator-app/blob/v2.0.0/changelog/23.x_32.x.md.).
- [prometheus-operator-crd](https://github.com/giantswarm/prometheus-operator-crd) version [v2.0.1](https://github.com/giantswarm/prometheus-operator-crd/releases/tag/v2.0.1). Update CRD for prometheus-operator.
- [security-pack](https://github.com/giantswarm/security-pack) version [0.9.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#090---2022-11-10) includes the Falco, Kyverno, Trivy, and Trivy Operator versions and associated improvements included in this announcement.
- [trivy-app](https://github.com/giantswarm/trivy-app/) version [0.7.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#070---2022-11-08) renames the chart to help standardize services names when deploying as part of the security pack.
- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.2.1](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#021---2022-11-08) changes the `trivy` server URL to standardize the service name when deploying as part of the security pack.

```